examples/mouse_picking.rs

```diff
@@ -1,6 +1,10 @@
 use bevy::prelude::*;
```

Comment:
This is a really cool demo, but I'd like to keep the `mouse_picking` example as minimal as possible to make it clear what the absolute minimum is required to get it working.

However, this updated example shows how to accomplish something a bit more complicated and representative. Let's keep the old version of `mouse_picking` as-is, and rename what you've created here to `multiple_sources`.

Reply:
Done